web/cypress/support/commands/utility-commands.ts

```diff
@@ -57,20 +57,38 @@ Cypress.Commands.add('waitUntilWithCustomTimeout', (
 
   Cypress.Commands.add('changeNamespace', (namespace: string) => {
     cy.log('Changing Namespace to: ' + namespace);
-    cy.byLegacyTestID('namespace-bar-dropdown').find('button').scrollIntoView().should('be.visible').click();
-    cy.get('[data-test="showSystemSwitch"]').then(($element)=> {
-      if ($element.attr('data-checked-state') !== 'true') {
-        cy.byTestID('showSystemSwitch').siblings('span').eq(0).should('be.visible').click();
+    cy.wait(2000);
+    cy.get('body').then(($body) => {
+      const hasNamespaceBarDropdown = $body.find('[data-test-id="namespace-bar-dropdown"]').length > 0;
+      if (hasNamespaceBarDropdown) {
+        cy.byLegacyTestID('namespace-bar-dropdown').find('button').scrollIntoView().should('be.visible');
+        cy.byLegacyTestID('namespace-bar-dropdown').find('button').scrollIntoView().should('be.visible').click({force: true});
+      } else {
+        cy.byClass('pf-v6-c-menu-toggle co-namespace-dropdown__menu-toggle').scrollIntoView().should('be.visible');
```

Comment:
maybe we can remove the pf specific class matcher or add a new test id so the tests work when backported.